src/azul/service/responseobjects/storage_service.py

```diff
@@ -158,6 +158,7 @@ def _upload_part(self, part):
         upload_part = self.mp_upload.Part(part.part_number)
         result = upload_part.upload(Body=part.content)
         part.etag = result['ETag']
+        part.content = None
```

Comment:
This should be in a separate commit titled: "Fix: multipart TSV upload leaks memory"